packages/datagrid/src/textrenderer.ts

```diff
@@ -205,13 +212,16 @@ class TextRenderer extends CellRenderer {
     // Compute the X position for the text.
     switch (hAlign) {
     case 'left':
-      textX = config.x + 2;
+      textX = config.x + 8;
```

Comment:
Why the change to the original padding? Should we make that configurable?

Reply:
Probably should be configurable. I found a slightly larger padding led to better white space balancing and made the grid much easier to scan and read when using it with real data.

Reply:
IIRC I got the original padding values from Excel on Windows.

Reply:
Ok, I'll change it back, or add it as an option.